gradio/utils.py

```diff
@@ -155,7 +148,11 @@ def alert_change(self):
         self.change_event.set()
 
     def swap_blocks(self, demo: Blocks):
+        old_blocks = self.running_app.blocks
         super().swap_blocks(demo)
```

Comment:
I think we need to take care of adding `max_file_size` from the old blocks here as well:

demo.max_file_size = old_blocks.max_file_size

Reply:
ok, handled in BaseReloader.swap_blocks